napari/utils/settings/_manager.py

```diff
@@ -210,4 +215,43 @@ def register_plugin(self, plugin):
         self._plugins.append(plugin)
 
 
-SETTINGS = SettingsManager()
+SETTINGS: Optional[SettingsManager] = None
```

Comment:
Make it a private variable to ensure that it should not be exposed (to prevent some failure in the future).

Reply:
Well, that would be a breaking change currently. We should do a deprecation cycle, but it is not urgent to start that process.

Reply:
>Well, that would be a breaking change currently

Yep exactly why it should remain there, it would continue to work the same way.

Reply:
But this PR alredy break this.

In every module which uses `from napari.utils.settings._manager import SETTINGS` the value for `SETTINGS` will be `None` 

To have working code it should be from napari.utils.settings import _manager` and called 
`_manager.SETTINGS`

Also in this or `napari/utils/settings/__init__.py` does not export SETTINGS object.

If you would like to kee backwart compatibility there should be a class:


```
class SettingsProxy:
	def __getattribute__(self, name):
		return getattr(get_settings(), name)
```
assigned to `SETTINGS` variable in this module.